tensorflow/lite/micro/tools/make/ext_libs/xtensa.inc

```diff
@@ -1,16 +1,33 @@
-ifeq ($(TARGET_ARCH), hifi4)
+ifeq ($(TARGET_ARCH), $(findstring $(TARGET_ARCH), "fusion_f1 hifi4"))
 
   DOWNLOAD_RESULT := $(shell $(MAKEFILE_DIR)/ext_libs/xtensa_download.sh ${MAKEFILE_DIR}/downloads hifi4)
   ifneq ($(DOWNLOAD_RESULT), SUCCESS)
     $(error Something went wrong with the xtensa download: $(DOWNLOAD_RESULT))
   endif
 
+  # TODO(b/161489252): -Wno-shadow is only needed for xannlib. But since we do
+  # not have separate cflags (or the concept of modular build targets) with the
+  # Makefile, -Wno-shadow will be used for everything.
+
+  PLATFORM_FLAGS = \
+    -DNNLIB_V2 \
+    -DXTENSA_NNLIB_MAX_SCRATCH_SIZE=70*1024 \
```

Comment:
Curious where 70k comes from. Is this due to hardware limitations?

Reply:
Unsure. Will discuss with @pnikam-cad separately. I have copied this from 
 https://github.com/tensorflow/tensorflow/blob/991b6592c1b59b4781ba0548cfab82ee55bc7a39/tensorflow/lite/micro/tools/make/ext_libs/xtensa_hifi_nn_library.inc#L9-L13

Filed http://b/176925218 to track this.

Reply:
This `-DXTENSA_NNLIB_MAX_SCRATCH_SIZE=70*1024` is coming from earlier implementation when we were using a stack buffer as scratch memory, as a temporary solution. 70 KB scratch size was needed for one of the example tests. 
We can remove this now, recently we have used the scratch allocation memory API on another branch. With that change the scratch memory will be allocated from Tensor Arena itself and we won't need this flag.